airflow/www/templates/airflow/dags.html

```diff
@@ -20,6 +20,51 @@
 {% extends base_template %}
 {% from 'appbuilder/loading_dots.html' import loading_dots %}
 
+{%- macro verbose_ordering_name(ordering) -%}
+    {{ 'descending' if ordering == 'desc' else ('ascending' if ordering == 'asc' else ordering) }}
+{%- endmacro -%}
+
+{%- macro sorted_th_link(name, attr_name) -%}
+   {% set request_ordering = (request.args.get('orderBy', 'unsorted')) %}
+   {% set curr_ordering = ('unsorted' if request.args.get('sortBy') != attr_name else request_ordering ) %}
+   {% set new_ordering = ('asc' if request.args.get('sortBy') != attr_name or curr_ordering != 'asc' else 'desc' ) %}
+   <a href="{{ url_for('Airflow.index',
+                       status=request.args.get('status', 'all'),
+                       search=request.args.get('search', None),
+                       tags=request.args.get('tags', None),
+                       sortBy=attr_name,
+                       orderBy=new_ordering
+                       ) }}">
+    {{ name }}
+    <span id="sorting-tip-{{ name }}" class="tooltip" role="tooltip" >
```

Comment:
You can simply this and circumvent adding this tooltip span by adding a `class="js-tooltip"` and a `title="{{ tooltip }}"` attributes to the parent anchor.

IMO, the current tooltip text is a bit too verbose. I would suggest dropping the "Column not currently sorted." and "Column sorted by…" all together and just provide description of what clicking on will do: "Sort by ascending owner" etc.

```jinja
<a
  …
  class="js-tooltip"
  title="Sort by {{ verbose_ordering_name(new_ordering) }} {{ attr_name }}"
>
```